src/main/java/com/wanderersoftherift/wotr/gui/screen/RuneAnvilScreen.java

```diff
@@ -40,6 +50,11 @@ public RuneAnvilScreen(RuneAnvilMenu menu, Inventory playerInventory, Component
     protected void init() {
         super.init();
 
+        fullDisplay = minecraft.screen.width > imageWidth + 2 * TOOLTIP_WIDTH_HINT + 4 * TOOLTIP_BORDER;
+        if (!fullDisplay) {
+            leftPos = Math.max(16, minecraft.screen.width - imageWidth - TOOLTIP_WIDTH_HINT - TOOLTIP_BORDER) / 2;
```

Comment:
I think it looks better like this:
```suggestion
            leftPos = Math.max(16, minecraft.screen.width - imageWidth - TOOLTIP_WIDTH_HINT - TOOLTIP_BORDER*2);
```